src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFSDriver.cs

```diff
@@ -0,0 +1,502 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Linq;
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    internal class SysFSDriver : UnixDriver
```

Comment:
Why changing this to be internal? I would either keep them public or expose a way from the InitUnixDriver method to pick which driver to use by force. This is to guard against the case of LibGpiodriver having a bug, so perhaps I want to intentionally use sysfs even though libgpiod is installed.

Reply:
That is what @joshfree suggested https://github.com/dotnet/iot/pull/133#discussion_r245388278